orttraining/orttraining/python/training/ortmodule/_utils.py

```diff
@@ -97,3 +97,10 @@ def _create_iobinding(io_binding, inputs, model, device):
 
     for value_info in model.graph.output:
         io_binding.bind_output(value_info.name, device.type, device_id=get_device_index(device))
+
+class ModuleMetadata():
```

Comment:
```suggestion
class _PytorchModuleMetadata():
```

We usually prepend internal classes with a _ to explicit inform it is a private class
A Pytorch prefix can also help indicating it is not ONNX metadata

Reply:
Since ```_utils``` was already internal, my thought was that the class needn't be prefixed by an ```_```. I'll make the changes though.